packages/vite/src/node/importGlob.ts

```diff
@@ -24,7 +26,8 @@ interface GlobParams {
   isAbsolute: boolean
 }
 
-export interface AssertOptions {
+interface GlobOptions {
+  as?: string
   assert?: {
     type: string
```

Comment:
I think we should add a JSDoc comment with `@deprecated`

Reply:
Good catch. Done.